client/.eslintrc.yml

```diff
@@ -23,6 +23,7 @@ rules:
   ### Stylistic issues
   indent: [1, 2, { SwitchCase: 1, VariableDeclarator: 2 }]
   id-length: [1, { min: 2, exceptions: [_, e, i, k, v] }]
+  space-before-function-paren: [1, "never"]
```

Comment:
In `eslint-config-airbnb@2.0.0` there was added [this rule](https://github.com/airbnb/javascript#7.11). Tbh I don't understand the reasoning. So I added one line to config, rather than add bunch of spaces all over the codebase for no good reason. Maybe we should create `eslint-config-shakacode`, which depends on `eslint-config-airbnb` + contains our rules that we want to change. What do you think about `space-before-function-paren` rule and `eslint-config-shakacode` idea?